client/src/app/site/pages/organization/pages/accounts/components/account-main/account-main.component.ts

```diff
@@ -30,16 +31,29 @@ export class AccountMainComponent extends BaseModelRequestHandlerComponent {
         modelRequestService: ModelRequestService,
         router: Router,
         openslidesRouter: OpenSlidesRouterService,
-        private controller: AccountCommonService
+        private controller: AccountCommonService,
+        private userController: UserControllerService
     ) {
         super(modelRequestService, router, openslidesRouter);
     }
 
     protected override async onBeforeModelRequests(): Promise<void> {
         this._accountIds = await this.controller.fetchAccountIds();
+        this.subscriptions.push(
+            this.userController.getViewModelListObservable().subscribe(async users => {
+                const userIds = users.map(user => user.id);
+                if (
+                    userIds.length !== this._accountIds.length ||
+                    !(userIds.sort().join(`,`) === this._accountIds.sort().join(`,`))
```

Comment:
`userIds.every((userId, idx) => userId === this._accountIds)` seems to be better because it breaks as soon as a difference is found (and does not unnecessarily convert it to strings). Also, saving `this._accountIds` in its sorted order saves time because the array does not have to be sorted newly on every update. Since the users per meeting can potentially be very large, it may be worth to save as much runtime as we can.

Reply:
Meinst du `userIds.every((userId, idx) => userId === this._accountIds[idx])`?

Reply:
yes, my bad